MakingItBetterG55/src/application/Main.java

```diff
@@ -3,20 +3,90 @@
 import javafx.application.Application;
 import javafx.stage.Stage;
 import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.ColumnConstraints;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.RowConstraints;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontWeight;
+import javafx.scene.text.Text;
 import javafx.scene.input.KeyCode;
 import javafx.scene.input.KeyEvent;
-import javafx.scene.layout.GridPane;
 import javafx.scene.layout.Pane;
 
-
 public class Main extends Application {
 	@Override
 	public void start(Stage primaryStage) {
 		try {
+			GridPane homeScreen = new GridPane();
+			
+			//Set this to your own URL or local directory for the background image file :)
+			homeScreen.setStyle("-fx-background-image: url('file:///C:/Users/Benjamin%20Keyes/Documents/JavaFX%20Workspace/MakingItBetter/src/images/woodbackground.jpg');");
```

Comment:
you can just use a relative file path, so if you have placed the image in src you should be able to write "/ImageName.fileExtenstion" see the ImageView in BoarderHandler for an example, that way it will work for everyone without changes

Reply:
Done.